src/clj/commiteth/db/core.clj

```diff
@@ -85,3 +84,11 @@
   (sql-value [value] (to-pg-json value))
   IPersistentVector
   (sql-value [value] (to-pg-json value)))
+
+(defmacro with-trx [& body]
```

Comment:
Like this macro!

Just I think that the abbreviation of transaction is normally `tx`.